packages/viz/__tests__/SizeCategoriesStyle.spec.ts

```diff
@@ -1,14 +1,9 @@
-import {
-  sizeCategoriesStyle,
-  defaultSizeCategoriesStyleOptions
-} from '../src/lib/style';
+import { sizeCategoriesStyle } from '../src/lib/style';
 
 describe('SizeCategoriesStyle', () => {
   describe('Style creation', () => {
     it('should create a sizeCategoriesStyle instance properly', () => {
-      expect(() =>
-        sizeCategoriesStyle('attributeName', defaultSizeCategoriesStyleOptions)
-      ).not.toThrow();
+      expect(() => sizeCategoriesStyle('attributeName')).not.toThrow();
```

Comment:
👍

Reply:
We should try to add more specs, regarding the proper use of bins / method names...

Reply:
Yes, agree. Working on that